### ISSUE TITLE
Modded Moons bugs

### ISSUE THREAD

**Issue by TheBrickBattla** [FluffyFishGames/AdvancedCompany#70]
**Describe the bug**
When launching AdvancedCompany, with modded moons, it struggles to add them to the terminal and show them, and ignoring one of my modded moons [secret labs] [secondary note, after deleting the mods with the moons [Generic Moons v4] it was still there in the moons category, even when relaunched several times, but secret labs never registered.]

**To Reproduce**
 how I personally reproduced it was simply launching the game with the mods, and getting in with it, and checking the Moons tab and having it state that theres moons I dont have anymore, or not showing a moon that I have [secret labs, generic moons] [generic moons was removed but still appears]
**Expected behavior**
What I expected to happen was really just to get in with some extra modded moons, nothing special really, maybe to fix it by removing the specific mods that could be causing it

**Screenshots**
[video instead https://medal.tv/games/lethal-company/clips/1SzE48ZoRpzj02/d1337rq5CPoB?invite=cr-MSxodlksMTgzNzE4MDc5LA ] if the link is too sketchy or you dont wanna take the risk, on medal search up TheBrickBattla, newest clip should be Bug Report, and thats just what it is

**Additional context**
Add any other context about the problem here. [i do not know if the modpack listed below will work properly and show the issue correctly because generic moons was removed from the modpack when I put the share code in, but it should still cause the effect. I will work on a fix myself first by un and re installing the mod] [and sorry for not knowing some stuff about logfiles, ect]

**Logfiles**
Please attach log files to your issue. If this is a multiplayer specific issue please provide logs of the host and the affected clients. [not multiplayer, affects both anyways and im unsure how to do this new to modding]

You can find the logs of the game in %appdata%/../LocalLow/ZeekerssRBLX/Lethal Company/

The log file is called "Player.log" (or just Player if you have file extensions hidden)

Please note: These logs will get overwritten everytime you start the game. So please get your logs right when the issue occured.

**Add a list of all of your mods**
modpack without generic moons [attempted fix, still looking at it] 018d6170-acbf-78c3-5806-c1bbdcc91f39]

**Comment by FluffyFishGames** [comment]
Can you please try 1.0.147? And please provide logs and full mod list.

**Comment by Qugar** [comment]
Seems like LC_API solves the problem, at least it doesn't get stuck for me anymore when I travel to modded moons.

LC_API: [https://thunderstore.io/c/lethal-company/p/2018/LC_API/](url)

**Comment by Qugar** [comment]
I suspect LC_API synchronizes the players regarding the modded moons, unfortunately I don't have much idea what exactly is happening.

Without Advanced Company the modded moons worked without any problems. When we installed Advanced Company there were endless loading times in the ship. I then only installed LC_API.

Whatever mod caused synchronization problems, that definitely doesn't have to be Advanced Company's fault, I just wanted to share my findings.

But thanks for the great mod, keep up the good work.
